README.md

```diff
@@ -1059,7 +1059,19 @@ being opened (for example, user accidental double tap), it will throw an Error
 with code `423` (Locked)
 
 There are some specific cases of bottom sheet, and we have some utility methods
-to make them simpler to use:
+to make them simpler to use: `bottomSheetSingleSelector`,
+`bottomSheetActionSelector` and `bottomSheetInfo`.
+
+⚠️ AVOID USING THESE METHODS ⚠️  
+There is a planned
+[BREAKING CHANGE: remove sheet specific methods](https://github.com/Telefonica/webview-bridge/pull/115)
+that will remove them from the webview-bridge library in the next major
+version.  
+The **bottomsheet** method should be used as it can be used to create all the
+specific cases.
+
+❗️IMPORTANT: If you are using Mistica in your web, prefer using Mistica methods
+instead. See https://github.com/Telefonica/mistica-web/blob/master/doc/sheet.md
```

Comment:
This is outdated, the README in master already has a notice about this:

https://github.com/Telefonica/webview-bridge?tab=readme-ov-file#bottomsheet

<img width="937" height="259" alt="image" src="https://github.com/user-attachments/assets/2953f833-a4d0-4c95-914a-3119ee3ebb6a" />